falcon/errors.py

```diff
@@ -214,7 +225,13 @@ class HTTPBadRequest(HTTPError):
     """
 
     @deprecated_args(allowed_positional=0)
-    def __init__(self, title=None, description=None, headers=None, **kwargs):
```

Comment:
Not strictly related to this PR, but we need to make a decision whether we want to follow through this deprecation and disallow positional `HTTPError` arguments in Falcon 4.0, or actually make the breaking change in 5.0.